server/sonar-server/src/test/java/org/sonar/server/user/index/UserIndexerTest.java

```diff
@@ -67,18 +69,39 @@ public void index_everything() {
   }
 
   @Test
-  public void index_single_user() {
+  public void index_single_user_on_startup() {
     UserDto user = db.users().insertUser();
 
-    UserIndexer indexer = createIndexer();
-    indexer.indexOnStartup(null);
+    underTest.indexOnStartup(null);
 
     List<UserDoc> docs = es.getDocuments(UserIndexDefinition.INDEX_TYPE_USER, UserDoc.class);
     assertThat(docs).hasSize(1);
     assertThat(docs).extracting(UserDoc::login).containsExactly(user.getLogin());
   }
 
-  private UserIndexer createIndexer() {
-    return new UserIndexer(db.getDbClient(), es.client());
+  @Test
+  public void index_single_user() {
+    UserDto user = db.users().insertUser();
+    UserDto anotherUser = db.users().insertUser();
+
+    underTest.index(user.getLogin());
+
+    List<UserDoc> docs = es.getDocuments(UserIndexDefinition.INDEX_TYPE_USER, UserDoc.class);
```

Comment:
As this method will probably disappear someday, it would be best to replace it by an es search, for instance : 
```
    assertThat(
      esTester.client()
        .prepareSearch(INDEX_TYPE_USER)
        .setQuery(termQuery(FIELD_ORGANIZATIONS, nameQuery))
        .get()
        .getHits()
        .getTotalHits()).isEqualTo(1);
```